packages/neon_framework/lib/src/blocs/unified_search.dart

```diff
@@ -188,7 +188,7 @@ class _UnifiedSearchBloc extends InteractiveBloc implements UnifiedSearchBloc {
   }
 
   bool providerMatchesApp(String providerID, AppImplementation app) =>
-      providerID == app.id || providerID.startsWith('${app.id}_');
+      providerID == app.id || providerID.startsWith('${app.id}_') || providerID.startsWith('${app.id}-');
```

Comment:
Do you think a regex would be easier to read?

Reply:
I don't think so and regex is likely heavier to compute than these simple string interpolations and comparisons.